.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/app-prototype
+    docker:
+      - image: circleci/ruby:2.4.2-node-browsers
+        environment:
+          PGHOST: 127.0.0.1
+          PGUSER: postgres
+          RAILS_ENV: test
+      - image: circleci/postgres:9.6.5-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: app_prototype_test
+          POSTGRES_PASSWORD: ""
+    steps:
+      - checkout
+
+      # Restore bundle cache
+      - type: cache-restore
+        key: bundle-{{ arch }}
```

Comment:
We have this so that the cache is busted when gems update:

`key: bundle-{{ arch }}-{{ checksum "Gemfile.lock" }}`